docs/index.rst

```diff
@@ -156,6 +156,17 @@ If you are using PyVista in your scientific research, please help our scientific
 visibility by citing our work! Head over to :ref:`citation_ref` to learn more
 about citing PyVista.
 
+Videos
+======
+
+Here are some videos that you can watch to learn pyvista:
+
+- SPyConJP2020 talk "How to plot unstructured mesh file on Jupyter Notebook" (15 minutes):
```

Comment:
Is it `SPyConJP2020` or `PyConJP2020` ? (without the 'S')

Reply:
Oops. I had typo. Thanks :+1:
```suggestion
- PyConJP2020 talk "How to plot unstructured mesh file on Jupyter Notebook" (15 minutes):
```